### PR TITLE
Clear all categories button

### PR DESCRIPTION
**Pull request by FMA126**
## Description

Fixes #

Add button to category form.  This button will clear the checkboxes in the category form.

## Motivation and Context

Previously, users had to manually uncheck all the category boxes.  This button makes it easier to clear the boxes and return the filtered resource to the default.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by timtheguy** [comment]
Thanks for the contribution. Can you clarify what you're adding by updating the pull request description and adding a relevant issue?

File: src/components/AdminPage/CategoryList.js [hunk 1]
```diff
@@ -25,18 +28,41 @@ export class CategoryList extends Component {
     this.props.actions.filterByCategories(selectedCategory.length > 0 ? filteredResource : this.props.resource);
   }
 
+  clearCategoryMenuItems() {
+    // this.categoryMenuItems().map
```
Comment (RajChaudhry) [inline src/components/AdminPage/CategoryList.js]: maybe uncomment this, and then the functionality will work?

File: src/components/AdminPage/CategoryList.js [hunk 2]
```diff
@@ -25,18 +28,41 @@ export class CategoryList extends Component {
...
   categoryMenuItems() {
     return this.props.categories.map((cat) =>
       <FormGroup key={cat} check>
         <Input type="checkbox" key={cat} onChange={() => this.handleChange(cat)} />{cat}
       </FormGroup>);
   }
+
+  handleClick = (event) => {
+    event.preventDefault();
+    console.log('hi');
```
Comment (RajChaudhry) [inline src/components/AdminPage/CategoryList.js]: I would take out all the testing console messages, if you want to keep any console messages, they should be descriptive, and relevant (ex: console message saying that all the categories are loaded)

**Comment by FMA126** [comment]
Just pushed to this again.  The form now clears.  However, the resources do not update when the form clears.

**Pull request closed without merging**